### DISCUSSION TITLE
Forward secrecy

### DISCUSSION THREAD

**Discussion by steida** [openpgpjs/openpgpjs#1497] [Q&A] [answered]
Do OpenPGP.js plan to support forward secrecy? Thank you

**Comment by twiss** [comment, marked as answer]
Hello :wave: We would like to, however, OpenPGP.js is an implementation of the OpenPGP specification. To implement forward secrecy, it would first need to be specified in the OpenPGP spec, so this might still take a while.
That being said, you could implement a weak form of forward secrecy by adding short-lived subkeys (and deleting the subkeys after decrypting the messages).
